netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -358,6 +469,82 @@ _net_ebpf_sock_addr_update_store_entries()
     NET_EBPF_EXT_RETURN_NTSTATUS(status);
 }
 
+static NTSTATUS
+_net_ebpf_sock_addr_create_security_descriptor()
+{
+    ACL* dacl = NULL;
```

Comment:
If its a static SD, why not hardcode the byte blob after you create it in your dev box?

Reply:
I had tried that (hard coded byte blob generated from user mode), but for some reason it did not work.